Game/Source/GDKTestGyms/PlayerDisconnectGameInstance.h

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "GDKTestGymsGameInstance.h"
+
+#include "PlayerDisconnectGameInstance.generated.h"
+
+UCLASS()
+class GDKTESTGYMS_API UPlayerDisconnectGameInstance : public UGDKTestGymsGameInstance
```

Comment:
I see in the test steps that it requires the tester to change to this game instance subclass to run the test. I think it'd be better to get rid of that step (anything that requires project settings changes is cumbersome for QA), and instead add this function to e.g. the player controller class for this gym.
That'd require a C++ version of the player controller class, in which case you can also get rid of the blueprint version of it and do the keybinding in C++ as well. Means we have one less asset that could run into engine version saving issues.

Reply:
Good point. I've removed the Player Controller BP and created it in C++ instead. This did give some unrelated trouble with anonymous namespaces. I have also removed the Game Instance so no extra test steps are needed